tests/unit/test_imbalance.py

```diff
@@ -19,11 +19,11 @@ def test_result_tfs(_tfs_file):
     res = merge_kmod_results.get_lumi_imbalance(_tfs_file)
 
     assert res["imbalance"] == 0.974139299943968
-    assert res["relative_error"] == 0.003859317636164786
-    assert res["eff_beta_ip1"] == 0.39800399369855577
-    assert res["rel_error_ip1"] == 0.0015434779687894644
-    assert res["eff_beta_ip5"] == 0.4085698972636139
-    assert res["rel_error_ip5"] == 0.0023158396673753213
+    assert res["relative_error"] == 0.0027110906825478205
```

Comment:
did you check if this is compatible with the formula?

Reply:
![image](https://user-images.githubusercontent.com/26184899/94687720-ec219400-032c-11eb-8c9b-297304a7aa05.png)

top: full error calculated by uncertainty package
bottom: full error calculated by formular

Reply:
So you mean that uncertainties might not be needed here?